.env.example

```
@@ -1,3 +1,28 @@
-VITE_SUPABASE_URL=https://your-project-ref.supabase.co
-VITE_SUPABASE_ANON_KEY=your-anon-key-here
-VITE_WTCS_GUILD_ID=
+# ─── Client build-time (VITE_* prefix — exposed to browser bundle) ───
+VITE_SUPABASE_URL=
+VITE_SUPABASE_ANON_KEY=
+VITE_SENTRY_DSN=
+VITE_POSTHOG_KEY=
+VITE_WTCS_GUILD_ID=         # WT esports Discord server ID (client-side guild membership check)
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)